Gems/AtomLyIntegration/CommonFeatures/Code/Source/Material/MaterialComponentController.cpp

```diff
@@ -130,6 +131,7 @@ namespace AZ
 
         void MaterialComponentController::Deactivate()
         {
+            AZ::Data::AssetBus::MultiHandler::BusDisconnect();
```

Comment:
Deactivate calls ReleaseMaterials three lines below this.

        void MaterialComponentController::ReleaseMaterials()
        {
            SystemTickBus::Handler::BusDisconnect();
            Data::AssetBus::MultiHandler::BusDisconnect();

Reply:
Yeah, I got confused between using two computers (at different commits) and my redundant call using`AZ::`, and my text search was using `AZ::*`, etc. All good now and I removed the redundant BusDisconnect.